compiler/src/ammtoaga.ts

```diff
@@ -135,9 +135,28 @@ const getHandlersMem = (handlers: LPNode[]) => handlers
     return handlerMem
   })
 
-const closuresFromDeclaration = (declaration: LPNode, closureMem: object, eventDecs: object) => {
+const closuresFromDeclaration = (
+  declaration: LPNode,
+  closureMem: object,
+  eventDecs: object,
+  addressMap: object,
+  argRerefOffset: number,
```

Comment:
can we leave a short comment about how argRerefOffset works?

Reply:
Sure